htdocs/takepos/pay.php

```diff
@@ -314,7 +521,11 @@ function ValidateSumup() {
 	),
 );
 $numpad = $conf->global->TAKEPOS_NUMPAD;
-
+if (!empty($conf->stripe->enabled) && !empty($conf->global->STRIPE_CARD_PRESENT)) {
```

Comment:
Same here: If "CASHDESK_ID_BANKACCOUNT_STRIPETERMINAL".$terminaltouse is not set, we should not output the message.